Update README for more precise Setup and Configuration (#150)
syssi/homeassistant-goecharger-mqtt: README.md

This is a custom component for Home Assistant to integrate the go-eCharger HOME+ and HOMEfix using the MQTT API (v2).

![Lovelace entities card](assets/lovelace-entities-card.png "Lovelace entities card")

## Setup and Configuration

> 1. Use HACS to install this custom component.

Reviewer: ```suggestion
1. Use HACS to install this custom component
```

1. Restart HA

> 1. Make sure you have an MQTT Broker ([e.g. Mosquitto broker](https://github.com/eclipse/mosquitto)) and a working [MQTT integration](https://www.home-assistant.io/integrations/mqtt/) in HA.

Reviewer: ```suggestion
1. Make sure you have an MQTT Broker ([e.g. Mosquitto broker](https://github.com/eclipse/mosquitto)) and a working [MQTT integration](https://www.home-assistant.io/integrations/mqtt/) in HA
```

1. (Recommended) Access to messages on MQTT Broker via client such as [MQTT Explorer](https://mqtt-explorer.com/)  

> 1. Enable MQTT access in the [go-eCharger App](https://play.google.com/store/apps/details?id=co.goe.app&hl=de)

Reviewer: ```suggestion
1. Enable the MQTT API of your charger at the [go-eCharger App](https://play.google.com/store/apps/details?id=co.goe.app&hl=de)
```

    - `Internet` -> `Advanced settings` -> scroll to bottom -> `Enable MQTT API` ([See also here](https://github.com/goecharger/go-eCharger-API-v2/blob/main/mqtt-en.md))

>     - in the field `URL` enter your MQTT broker URL, e.g. mqtt://192.168.178.3:1883 for a broker running on a server with IP 192.168.178.3 and with default port 1883

Reviewer: ```suggestion
    - in the field `URL` enter your MQTT broker URL, e.g. mqtt://192.168.178.3:1883 for a broker running on a server with IP 192.168.178.3 on default port 1883
```

>     - in the field `URL` enter your MQTT broker URL, e.g. mqtt://192.168.178.3:1883 for a broker running on a server with IP 192.168.178.3 on default port 1883

Reviewer: ```suggestion
    - at the field `URL` enter your MQTT broker URL, e.g. mqtt://192.168.178.3:1883 for a broker running on a server with IP 192.168.178.3 on default port 1883
```

    ![alt text](assets/goe_app_settings.jpg)
    - (optional) Add MQTT keys and certificates as needed
    - Press the `SAVE` symbol in the top right to apply the settings

> 1. (Optional) Make sure you see a new topic on the MQTT broker using your MQTT client.

Reviewer: ```suggestion
1. (Optional) Make sure you see a new topic on the MQTT broker using your MQTT client
```
